src/graphql/resolvers/query.ts

```diff
@@ -1,9 +1,6 @@
-import { byCategory } from '../models/jokes';
+import { QueryResolvers } from '../_generated/types';
 
-const resolvers = {
-  Query: {
-    jokes: () => ({}),
-  },
+export const query: QueryResolvers.Type = {
+  ...QueryResolvers.defaultResolvers,
+  jokes: () => ({}),
```

Comment:
can we omit this resolver and keep `defaultResolvers` ?

Reply:
No, it needs to be implemented because of `QueryResolvers.Type` interface. Previously there also was this stub.

Reply:
ok, cool